components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -641,6 +645,11 @@ public String getDeviceAuthzEPUrl() {
 
         return deviceAuthzEPUrl;
     }
+    
+    public boolean isAddTenantDomainToTokenEnabled() {
+        return addTenantDomainToTokenEnabled;
```

Comment:
Add a new line after the method definition. Check other places as well.

Reply:
fixed with [5ec7232](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2384/commits/5ec7232e795511b0db13a38fe8c7dbf9a27cdaaa)